app/components/concept.ts

```diff
@@ -168,4 +191,11 @@ export default class ConceptComponent extends Component<Signature> {
       window.history.replaceState({}, '', `${window.location.pathname}?${urlParams.toString()}`);
     }
   }
+
+  updateProgressPercentage(progressPercentage: number) {
+    if (progressPercentage > this.args.latestConceptEngagement.currentProgressPercentage) {
```

Comment:
@libmartinito do you think we need this check? If a user clicks on "step back" on a concept a couple of times, maybe they expect to be taken back to where they were? Would everything work fine if we removed this check and allowed `progressPercentage` to decrease if a user clicks on "step back"?